src/osd/scrubber_common.h

```diff
@@ -29,6 +47,25 @@ struct ScrubPreconds {
   bool only_deadlined{false};
 };
 
+/// PG services used by the scrubber backend
+struct PgScrubBeListener {
+  virtual ~PgScrubBeListener() = default;
+
+  virtual const PGPool& get_pgpool() const = 0;
```

Comment:
Why not name this get_pool()?  I don't think it would be ambiguous.

Reply:
For a time - it was, as I had structures that provided both APIS, i.e. both types of get_pool().
Seems that it's no longer so, but still I think the change makes sense: otherwise -
it is hardly evident from a piece of code what interface is used.
- and the number of actual callers that had to be changed is small.

Reply:
Ok, we can change it to get_pgpool() if there's a get_pool somewhere that returns a PGPool&.

Reply:
in PG.h it's:
`const PGPool& get_pool() const`

but in PrimaryLogPG (that inherits from it - but also from PGBackend::listener) it's
`const pg_pool_t &get_pool() const override`

Reply:
Oh!  Yeah, I'm with you, PG::get_pgpool() it is.